src/axel.c

```diff
@@ -846,9 +849,8 @@ axel_divide(axel_t *axel)
 		axel->conn[i].currentbyte = axel->conn[i - 1].lastbyte + 1;
 		axel->conn[i].lastbyte =
 		    axel->conn[i].currentbyte +
-		    axel->size / axel->conf->num_connections;
+		    (axel->size / axel->conf->num_connections) - 1;
 	}
-	axel->conn[axel->conf->num_connections - 1].lastbyte = axel->size - 1;
```

Comment:
Instead of removing this, it should add `size % num_connections` here. That way the loop is generic and can fill all items.